packages/js/src/axiomClient.ts

```diff
@@ -0,0 +1,260 @@
+import { Axiom } from "./client";
+import { ClientOptions } from "./httpClient";
+
+
+interface LogEvent {
+  level: string;
+  message: string;
+  fields: any;
+  _time: string; 
+}
+
+enum LogLevel {
+  debug = 0,
+  info = 1,
+  warn = 2,
+  error = 3,
+  off = 100,
+}
+
+export type LoggerConfig = {
+  args?: { [key: string]: any };
+  logLevel?: LogLevel;
+  autoFlush?: boolean;
+  source?: string;
+  req?: any;
+  token: string | undefined;
+  dataset: string | undefined;
+};
+
+
+const Version = 'AXIOM_VERSION';
+const LOG_LEVEL = process.env.AXIOM_LOG_LEVEL || 'debug';
+
+const isBrowser = typeof window !== 'undefined';
+const isVercel = process.env.NEXT_PUBLIC_AXIOM_INGEST_ENDPOINT || process.env.AXIOM_INGEST_ENDPOINT;
+const isNoPrettyPrint = process.env.AXIOM_NO_PRETTY_PRINT == 'true' ? true : false;
+
+function buildLogsEndpoint(dataset: string) {
+    const axiomUrl = 'https://api.axiom.co';
+    return `${axiomUrl}/api/v1/datasets/${dataset}/ingest`;
+  }
+
+export class AxiomClient extends Axiom  {
```

Comment:
can you put the axiom client inside Axiom? aka add the `info`, `log`, etc. methods to the main client and make it work